web/src/main/java/com/graphhopper/srlarray/SpatialRuleLookupArray.java

```diff
@@ -115,7 +131,7 @@ public SpatialRule lookupRule(double lat, double lon) {
         return SpatialRule.EMPTY;
     }
 
-    protected int getRuleContainerIndex(int xIndex, int yIndex) {
+    private int getRuleContainerIndex(int xIndex, int yIndex) {
```

Comment:
Because it's possible. Less public methods -> better. (protected is almost public)